src/backend/partaj/core/api/referral.py

```diff
@@ -364,3 +364,54 @@ def unassign_unit(self, request, pk):
             )
 
         return Response(data=ReferralSerializer(referral).data)
+
+    @action(
+        detail=True, methods=["post"], permission_classes=[UserIsReferralUnitOrganizer]
+    )
+    # pylint: disable=invalid-name
+    def change_urgencylevel(self, request, pk):
+        """
+        change referral's urgency level
+        """
+
+        # check explanation not empty
+        if not request.POST.get("urgencylevel_explanation"):
```

Comment:
I'd suggest you always use `request.data.get` to access post data. Here you are using 3 different methods to access the same data (`request.POST.get`, `request.data.get` `request.data['value']`)